cider/exec/module/CiderRuntimeModule.cpp

```diff
@@ -742,10 +742,16 @@ void CiderRuntimeModule::initCiderAggTargetColExtractors() {
         CiderAggTargetColExtractorBuilder::buildCiderAggTargetColExtractor(
             group_by_agg_hashtable_.get(), target_col_index, is_partial_avg_sum[i]);
 
-    // TODO: Partial Avg
     if (kAVG == col_info.agg_type) {
```

Comment:
For partial AVG, the `col_info.agg_type` should still be `kSUM `and `kCount`?

Reply:
No, it's not correct, I think I misunderstood before. Still working on another PR and added WIP label..